packages/lesswrong/server/codegen/generateDbTypes.ts

```diff
@@ -36,16 +37,8 @@ function generateCollectionDbType(collection: CollectionBase<any>): string {
   
   for (let fieldName of Object.keys(schema)) {
     const fieldSchema = schema[fieldName];
-    // Resolver-only field?
```

Comment:
This is just a hackier version of the `isResolverOnly` function that we've replaced it with below